src/components/GradeItemAccordion.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import TextInput from './TextInput';
+import DelButton from './MainActionButton';
+import { CourseItem, Fraction } from '../api';
```

Comment:
So I think you can actually change this import to:
```suggestion
import { CourseItem, Fraction } from 'api';
```

If this doesn't work though, you can omit this change. I tried to set everything up though so we can import from the topdown if things are in different primary folders.